release-content/0.14/release-notes/new_math_primitives.md

```diff
@@ -0,0 +1,15 @@
+Geometric shapes find a variety of applications in game development, ranging from rendering simple items to the screen for display / debugging to use in
+colliders, physics, raycasting, and more.
+
+For this, geometric shape primitives were introduced in Bevy 0.13, and work on this area has continued with Bevy 0.14, which brings the addition of 
```

Comment:
```suggestion
For this, geometric shape primitives were [introduced in Bevy 0.13](https://bevyengine.org/news/bevy-0-13/#primitive-shapes), and work on this area has continued with Bevy 0.14, which brings the addition of 
```

Maybe a link to the introductory blog post would be beneficial? Other than that, LGTM and thanks for taking the time to write the release notes! ^^